src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Instantiate make random class level field (class Random should be instantiated once)
```suggestion
        int index = random.nextInt(Color.values().length);
```